master/buildbot/reporters/mail.py

```diff
@@ -43,6 +43,11 @@
 from buildbot.reporters.message import MessageFormatter as DefaultMessageFormatter
 from buildbot.util import service
 
+if PY3:
```

Comment:
just to make it a bit more specific (it did not get to our [meetings minutes](https://supybot.buildbot.net/meetings/buildbot/2016/buildbot.2016-11-15-16.29.html):

```
> 19:11 <@djmitche> https://github.com/edschofield maybe?
```